File: backend/configs/urls.py
from django.urls import path, include


urlpatterns = [
    path('api/auth', include('apps.auth.urls')),
    path('api/products', include('apps.products.urls')),
    path('api/categories', include('apps.categories.urls')),
    path('api/orders', include('apps.orders.urls')),
]

File: frontend/src/constants/urls.js
const baseURL = '/api'

const auth = '/auth'
const register = '/register'
const products = '/products'
const categories = '/categories'
const orders = '/orders'

const urls = {
    auth: {
        login: auth,
        register: `${auth}${register}`
    },
    products,
    categories,
    orders
}

export {
    baseURL,
    urls
}